src/libs/CardUtils.ts

```diff
@@ -462,18 +463,17 @@ function checkIfNewFeedConnected(prevFeedsData: CompanyFeeds, currentFeedsData:
     };
 }
 
-function filterClosedCards(cards: CardList | undefined): CardList {
+function filterInactiveCards(cards: CardList | undefined): CardList {
     const closedStates: number[] = [CONST.EXPENSIFY_CARD.STATE.CLOSED, CONST.EXPENSIFY_CARD.STATE.STATE_DEACTIVATED, CONST.EXPENSIFY_CARD.STATE.STATE_SUSPENDED];
-    const filteredCards = Object.entries(cards ?? {}).filter(([, card]) => !closedStates.includes(card.state));
-    return Object.fromEntries(filteredCards);
+    return filterObject(cards ?? {}, (key: string, card: Card) => !closedStates.includes(card.state));
```

Comment:
```suggestion
    return filterObject(cards ?? {}, (key, card) => !closedStates.includes(card.state));
```
NAB. We should keep let the types to be picked by ts